sofar/sofastream.py

```diff
@@ -71,6 +53,8 @@ class SofaStream():
         >>>     specific_irs = data[:,0,:]
         >>>     print(specific_irs.shape)
         (11950, 256)
+
+    If you want to use all the data from a SOFA-file use :class:`SOFA` instead.
```

Comment:
Maybe move this hint up right before the Parameters section and add a corresponding hint to the `Sofa` class?